outlines/using-packages.md

```diff
@@ -0,0 +1,34 @@
+# Using packages
+
+1. Packages vs modules
+  - Packages are useful for sharing content between apps and users
+  - It's usually better to leverage someone else's work and use a package if it does what you need
+2. Npm vs Atmosphere
+  1. Npm packages:
```

Comment:
I guess I thought I might detail why we have both and why you might choose one or the other before getting into brass tacks